src/tests/ipc-handlers.test.ts

```diff
@@ -31,7 +31,10 @@ const mockUserData = {
 	isDirectory: () => true,
 } );
 
-const mockIpcMainInvokeEvent = {} as IpcMainInvokeEvent;
+const mockIpcMainInvokeEvent = {
+	sender: { isDestroyed: jest.fn( () => false ) },
+	// Double assert the type with `unknown` to simplify mocking this value
+} as unknown as IpcMainInvokeEvent;
```

Comment:
This "fix" was provided via Copilot's prompt. While using `unknown` feels very similar to `any` (which our linter discourages), using "double assertion"[^1][^2] felt appropriate for a test mock. 

[^1]: [Typescript: Why does `as unknown as x` work](https://stackoverflow.com/a/69399373/378228)
[^2]: [Understanding "as unknown as" in TypeScript](https://www.omarileon.me/blog/as-unknown-as-typescript)